components/Dashboard/Nav/Nav.tsx

```diff
@@ -32,14 +49,77 @@ const Nav: React.FC<Props> = ({ expanded, collapse }) => {
     }
   }
 
+  const handleLogOut = (): void => {
+    handleCollapse()
+    logout()
+
+    // Clear the currentUser cache
+    refetch()
+
+    // Redirect to home page
+    Router.push('/')
+  }
+
```

Comment:
**NICE** thanks for fixing this!!

Reply:
Ah, @Lanny I just tested logout on staging and I'm seeing the same behavior as before. Super happy to get the nav fixed on settings page but just checking if this code was intended to also fix logout?

Reply:
It was not, this is probably the piece of code to look at but this PR was just trying to get the nav working on the settings page.

Reply:
Ah ok cool will merge now then! 😄